doc-changelog/action.yml

```diff
@@ -100,6 +100,15 @@ inputs:
     default: false
     type: boolean
 
+  use-conventional-commits: # TODO: Remove deprecated input in v11
```

Comment:
Deprecation would have to happen in v12 I believe. On v10 this should throw a warning. On v11, an error. On v12 (or even later) we can completely remove the input.

Reply:
I'm saying this because I don't know if your plan is to have this PR released ASAP. If that's the case it would have to go into v10... and we shouldn't break things. Thus, the warning.

Reply:
Done.